health-services/individual/src/test/java/org/egov/individual/validator/AddressTypeValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package org.egov.individual.validator;
+
+import lombok.extern.slf4j.Slf4j;
+import org.egov.common.models.Error;
+import org.egov.common.models.individual.Address;
+import org.egov.common.models.individual.AddressType;
+import org.egov.common.models.individual.Individual;
+import org.egov.common.models.individual.IndividualBulkRequest;
+import org.egov.individual.helper.IndividualBulkRequestTestBuilder;
+import org.egov.individual.helper.IndividualTestBuilder;
+import org.egov.individual.validators.AddressTypeValidator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(MockitoExtension.class)
+@Slf4j
+public class AddressTypeValidatorTest {
+
+    @InjectMocks
+    private AddressTypeValidator addressTypeValidator;
+
+    @Test
+    void shouldNotGiveError_WhenAddressTypeIsValid() {
```

Comment:
Follow the naming convention. Remove underscore please!

Reply:
Removed from this class and other classes